jujugui/static/gui/src/app/init/bundle-importer.js

```diff
@@ -484,7 +484,8 @@ class BundleImporter {
           constraints: constraints
         });
       }
-      this.modelAPI.update_annotations(name, 'application', annotations);
+      this.modelAPI.update_annotations(name, 'application', annotations,
+        {immediate: true});
```

Comment:
Why immediate true? Won't this run before the ECS has executed so if any changes are made to the canvas pre-deploy they wont' be reflected in this annotation.

Reply:
So that I can work in smaller bits of work. Future branch.